client/src/actions/index.js

```diff
@@ -19,6 +19,22 @@ export function fetchUser() {
   }
 }
 
+export function sendSearchKey(searchKey) {
```

Comment:
I think this would make more sense to be called `fetchResults`. That's what it's doing right?

Reply:
Yeah I can change this.